lib/assert/context.rb

```diff
@@ -92,8 +70,10 @@ def assert_not(assertion, fail_desc = nil)
     end
     alias_method :refute, :assert_not
 
-    def assert_that(actual_value)
-      Assert::ActualValue.new(actual_value, context: self)
+    def assert_that(
+          actual_value = Assert::ActualValue.not_given,
+          &actual_value_block)
```

Comment:
@kellyredding - I think we decided to wrap the `)` to the next line?

Reply:
@jcredding yes! thank you!